src/workerd/api/node/zlib-util.c++

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+#include "zlib-util.h"
+
+#include "zlib.h"
+
+namespace workerd::api::node {
+
+namespace {
+// This function is required to avoid function name clash.
+uint32_t crc32Impl(uint32_t value, unsigned char* bytes, size_t size) {
+  return crc32(value, reinterpret_cast<const Bytef*>(bytes), size);
+}
+} // namespace
+
+uint32_t ZlibUtil::crc32(kj::Array<kj::byte> data, uint32_t value) {
+  return crc32Impl(value, data.asBytes().begin(), data.size());
```

Comment:
nit: The "asBytes()" here is not necessary

Reply:
Thanks. Updating it now!